src/sentry/api/serializers/rest_framework/notification_action.py

```diff
@@ -43,16 +43,50 @@ class NotificationActionSerializer(CamelSnakeModelSerializer):
     Django Rest Framework serializer for incoming NotificationAction API payloads
     """
 
-    integration_id = serializers.IntegerField(required=False)
-    sentry_app_id = serializers.IntegerField(required=False)
-    projects = serializers.ListField(child=ProjectField(scope="project:read"), required=False)
-
-    service_type = serializers.CharField()
-    target_type = serializers.CharField()
-    trigger_type = serializers.CharField()
-
-    target_identifier = serializers.CharField(required=False)
-    target_display = serializers.CharField(required=False)
+    integration_id = serializers.IntegerField(
+        help_text="""ID of integration used as the notification service. For example, ID of
+            Slack integration if Slack is used to notify org members.
+            This field is required if **service_type** is `slack`, `pagerduty` or `opsgenie`.""",
```

Comment:
```suggestion
        help_text="""ID of the integration used as the notification service. For example, this would 
            be the Slack Integration ID if the action has Slack notify organization members.
            This field is required if **service_type** is `slack`, `pagerduty` or `opsgenie`.""",
```